fao-house-style.csl

```diff
@@ -0,0 +1,183 @@
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <names variable="editor"/>
+        <text variable="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="4" initialize-with="."/>
+      <et-al font-style="italic" prefix=" "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text variable="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="publisher">
+    <group prefix=" ">
+      <text variable="publisher-place" suffix=","/>
+      <text variable="publisher" prefix=" " suffix="."/>
+    </group>
+  </macro>
+  <macro name="edition">
+    <number font-style="italic" vertical-align="baseline" suffix=" " variable="edition" form="ordinal"/>
+    <text term="edition" font-style="italic" suffix=". "/>
+  </macro>
+  <macro name="Available-at-and-accessed">
+    <text term="available at" text-case="capitalize-first" vertical-align="baseline" suffix=": "/>
+    <text variable="URL" suffix=". "/>
+    <text term="accessed" text-case="capitalize-first" suffix=" "/>
+    <date form="text" variable="accessed" suffix="."/>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
+    <sort>
+      <key variable="author"/>
+      <key variable="issued"/>
+      <key variable="accessed"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
+        <text macro="author-short"/>
+        <choose>
+          <if match="any" variable="issued">
+            <date date-parts="year" form="text" variable="issued"/>
+          </if>
+          <else>
+            <text term="no date" form="short"/>
+          </else>
+        </choose>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="13" et-al-use-first="1" entry-spacing="0" hanging-indent="true">
+    <sort>
+      <key macro="author"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <text macro="author" font-weight="bold"/>
+      <choose>
+        <if variable="issued">
+          <date variable="issued" suffix=". ">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture song" match="any">
+          <choose>
+            <if match="any" is-numeric="edition">
+              <group delimiter=" " prefix=" ">
```

Comment:
way too complex here: Why not just wrap title and edition into a group with delimiter=", " and suffix=". " and remove the suffix=". " from the edition term above.